tests/sentry/grouping/test_categorization.py

```diff
@@ -59,7 +59,9 @@
 _fixture_path = os.path.join(os.path.dirname(__file__), "categorization_inputs")
 
 _SHOULD_DELETE_DATA = os.environ.get("SENTRY_TEST_GROUPING_DELETE_USELESS_DATA") == "1"
-_DELETE_KEYWORDS = os.environ.get("SENTRY_TEST_GROUPING_DELETE_KEYWORDS", "").lower().split(",")
+_DELETE_KEYWORDS = [
+    x for x in os.environ.get("SENTRY_TEST_GROUPING_DELETE_KEYWORDS", "").lower().split(",") if x
```

Comment:
for context, this fixes a bug where all snapshots were deleted on machines that did not have this environment variable set